profiles/common/modules/custom/multisite_drupal_features_set_standard/multisite_drupal_features_set_standard.info

```diff
@@ -398,3 +398,13 @@ feature_sets[multisite_business_indicators_standard][icon] = ''
 feature_sets[multisite_business_indicators_standard][font] = 'multisite-icon-stats'
 feature_sets[multisite_business_indicators_standard][enable][] = multisite_business_indicators_standard
 feature_sets[multisite_business_indicators_standard][disable][] = multisite_business_indicators_standard
+
+; NextEuropa Tracked Changes
+feature_sets[nexteuropa_trackedchanges][title] = 'WYSIWYG Tracked Changes'
+feature_sets[nexteuropa_trackedchanges][description] = 'Provides editorial teams with change tracking capabilities in WYSIWYG.'
+feature_sets[nexteuropa_trackedchanges][documentation] = 'https://webgate.ec.europa.eu/fpfis/wikis/x/ayXFCg'
+feature_sets[nexteuropa_trackedchanges][category] = 'Editorial Management'
+feature_sets[nexteuropa_trackedchanges][icon] = ''
+feature_sets[nexteuropa_trackedchanges][font] = 'multisite-icon-keyboard'
```

Comment:
@Fefaine, I 've chosen it because it expresses well to what it is related. Like GIS field and GeoJson feeds, it touches the same finterface as Maxlength.
but OK. I will see if I find another one.